plugins/org.wso2.developerstudio.eclipse.gmf.esb.persistence/src/org/wso2/developerstudio/eclipse/gmf/esb/internal/persistence/CacheMediatorTransformer.java

```diff
@@ -68,30 +70,31 @@ public void transformWithinSequence(TransformationInfo information,
 		 */
 		Assert.isTrue(subject instanceof CacheMediator, "Invalid subject.");
 		CacheMediator visualCache = (CacheMediator) subject;
-
 		/*
 		 *  Configure Cache mediator.
 		 */
 		org.wso2.carbon.mediator.cache.CacheMediator cacheMediator = new org.wso2.carbon.mediator.cache.CacheMediator();
 		setCommonProperties(cacheMediator, visualCache);
 		{	
-			if(visualCache.getCacheAction().getValue()==0){
-				if (StringUtils.isNotBlank(visualCache.getCacheId())) {
-					cacheMediator.setId(visualCache.getCacheId());
-				}			
-				cacheMediator.setScope(visualCache.getCacheScope().getLiteral());
-				cacheMediator.setTimeout(visualCache.getCacheTimeout());
+			if(visualCache.getCacheAction().getValue() == 0){				
```

Comment:
getValue method returns the value generated by the gmf model for that particular cache action. According to the generated gmf model, 0 represents the "FINDER" action. 
will check the formatting.